svc/FetchService.ts

```diff
@@ -286,6 +288,11 @@ export class FetchService extends HoistService {
         }
     }
 
+    private async jsonAsync(response: Response) {
```

Comment:
```suggestion
    // Cleanly return undefined for an empty response body (i.e. avoid throwing).
    // Successful 2xx HTTP status codes (e.g. 202 ACCEPTED) might or might not return a body.
    private async jsonAsync(response: Response) {
```

Reply:
Not sure about this.  In the past, I've gotten API devs to fix/change the returned status code to 204 when they insisted they could not return content.

I searched a couple http status docs sites and they did not suggest that content was optional for a 202.  Content was required.

Are we being flexible here to be nice?

Reply:
wow -- this is clever and hardcore.  

Wondering how this differs from the native `.json()` and the way it should be implemented according to the spec.

We could also consider trying .json() first, and falling back on this.

Reply:
also might call this `safeJsonAsync()` and reference in the comment that we are trying to reproduce a version of the built-in `.json()` that does not throw for empty bodies.....